float64[] init_joints
float64[] pose
---
float64[] targetJoints
